portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -30,21 +30,29 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8144 | Test Summary: Properties management menu is present"
-	@ignore = "true"
 	@priority = "5"
 	test CanNavigateToProperties {
+		ACUtils.launchAC();
 
-		// TODO LRAC-8144 Automate Properties#CanNavigateToProperties pending implementation
+		ACNavigation.goToSettings();
 
+		ACSettings.goToProperties();
+
+		ACProperties.assertPropertyTitle(propertyName = "Properties");
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8145 | Test Summary: Owner can create a new property"
-	@ignore = "true"
 	@priority = "5"
 	test OwnerCanCreateNewProperty {
+		ACUtils.launchAC();
```

Comment:
Hey @joshitagaki , I would suggest an improvement here to increase the test coverage...

I think should be good to validate if only Owners are able to create new Properties, for this, we need to check that non-Owners are not allowed to create new properties, so, we could perform the login with the user `corbin.murakami@faro.io` which is a simple user... and assert that "**add new property**" button is not present.

In this way, we will be covering everything, Owners are allowed - as you did with `test@liferay.com` user and now that simple Users are not allowed - using `corbin.murakami@faro.io` user.  I also noticed that admins (like the user `michelle.hoshi@faro.io`) are also able to create new properties, so it would be an option too.

What do you think?

Thanks!

Reply:
You probably will need to implement:

- a macro to perform the logout
- update the ACUtils#launchAC since we are always logging by default with the `test@liferay.com` user when calling it, so it should be updated to something like:
```
	macro launchAC {

		var analyticsCloudURL = PropsUtil.get("analytics.cloud.url");

		Open(locator1 = "${analyticsCloudURL}");

		if (!(isSet(emailAddress))) {
			emailAddress = "test@liferay.com";
		}

		if (!(isSet(password))) {
			password = "test";
		}

		if (IsElementPresent(locator1 = "ACLogin#EMAIL_LOGIN_FIELD")) {
			ACLogin.loginAs(
				emailAddress = "${emailAddress}",
				password = "${password}");
		}

		ACWorkspace.openWorkspace();
	}
```
where we need to use the same strategy we do on `ACLogin#loginAs`, having a way to get the email address and password from the test body that will be like:

`ACUtils.launchAC(emailAddress = "corbin.murakami@faro.io", password = "test");`

So it will continue working for the current tests that uses `test` user, and for the cases we would like to login with a different user, we pass it through the `launchAC` macro.

Reply:
I haven't tested this implementation, it is just a tip/suggestion. ;)